### ISSUE TITLE
GitHub executor does not wait for `version` to finish

### ISSUE THREAD

**Issue by LuckeeDev** [jscutlery/semver#461]
I have the following configuration:

```json
"version": {
	"executor": "@jscutlery/semver:version",
	"options": {
		"push": true,
		"noVerify": true,
		"postTargets": ["workspace:github"]
	}
},
"github": {
	"executor": "@jscutlery/semver:github",
	"options": {
		"tag": "${tag}",
		"notes": "${notes}",
		"title": "Release ${tag}"
	}
}
```

However, whenever I try to run `nx run workspace:version`, I get the following error:
```
tag vx.y.z exists locally but has not been pushed to REPONAME, please push it before continuing or specify the `--target` flag to create a new tag
```

It seems like `semver:version` does not wait for the GitHub push before finishing and calling the `postTargets`.

**Comment by edbzn** [comment]
Hi @LuckeeDev, by any chance can I access a repository to reproduce the issue?

**Comment by LuckeeDev** [comment]
Yes sure @edbzn, [here](https://github.com/LuckeeDev/csl/runs/5499099875?check_suite_focus=true) you can find the logs of a past GitHub action which failed due to this error. [This](https://github.com/LuckeeDev/csl/commit/1fe49a4c31783855b8d606d0d38bbb9b933978ee) is the commit associated with the Github action run. I also tried it locally in the same conditions but I got the same error.

**Comment by Brian-McBride** [comment]
Just ran across the same problem
